bbn-test-4/finality-providers/registry/hiya.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": {
+    "moniker": "hiya",
+    "identity": "",
+    "website": "",
+    "security_contact": "",
```

Comment:
Thanks! I would appreciate it if you could specify a contact and recreate the signature.

Reply:
Contact information has been added, please review the code again.